worker/provisioner/provisioner_test.go

```diff
@@ -1524,6 +1524,17 @@ func (s *ProvisionerSuite) TestProvisionerRetriesTransientErrors(c *gc.C) {
 	// instance has started.
 	thatsAllFolks := make(chan struct{})
```

Comment:
Worth renaming to _m3StartConfirmed_ or some such? I found I definitely had to read the rest of the test to assign meaning before going back and continuing to parse.

Reply:
I renamed them to runSetStatusGoroutine and setStatusGoroutineDone

Does that work better for you?